.github/workflows/build.yml

```diff
@@ -0,0 +1,50 @@
+name: "Linting and Testing"
+
+on:
+  push:
+    branches: [master,github-actions-test]
+    paths-ignore:
+    - 'README.md'
+    - '.deepsource.toml'
+    - '.gitignore'
+
+  pull_request:
+    branches: [master]
+    paths-ignore:
+    - 'README.md'
+    - '.deepsource.toml'
+    - '.gitignore'
+
+
+jobs:
+  build:
+
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        python-version: [3.6,3.7,3.8,3.9]
+        test-folder : ['tests.tests_iotools']
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build using Python ${{matrix.python-version}}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{matrix.python-version}}
+
+
+      - name: install dependencies
+        run:  |
+          # $CONDA is an environment variable pointing to the root of the miniconda directory
+          $CONDA/bin/conda env update --file iospi.yml --name base
+          $CONDA/bin/conda install flake8
```

Comment:
let's install flake8-black right away, instead of flake8 (different from what we do in geomstats): flake8 and black have different conventions: we might get weird errors if we don't use flake8-black:
https://pypi.org/project/flake8-black/

Reply:
Yes makes sense. I replaced, let's see how it works.